Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -35,9 +35,8 @@ import qualified Domain.Types.DriverOnboarding.VehicleRegistrationCertificate as
 import qualified Domain.Types.Merchant as DM
 import qualified Domain.Types.Person as Person
 import Environment
-import qualified Idfy.Flow as Idfy
-import qualified Idfy.Types as Idfy
 import Kernel.External.Encryption
+import qualified Kernel.External.Verification.Idfy.Types.Response as Idfy
```

Comment:
Why to expose Idfy Response type here? It should be common Verification types, right?

Reply:
@hkmangla that's because I use common `Verification` types only for sync requests. For async callbacks I still use `Idfy` types. If we need to merge this PR right now, then async types can be refactored in separate PR